lib/availability_resolver.js

```diff
@@ -58,19 +59,26 @@ class AvailabilityResolver {
       logger.debug('_fixItemStatusAggregation: Skipping because no SCSB statuses provided')
       return Promise.resolve()
     }
+    const bnum = this.elasticSearchResponse.hits.hits[0]._id
+    const { nyplSource } = NyplSourceMapper.instance().splitIdentifier(bnum)
 
-    // Count number of ReCAP items by summing the counts for rc* locations:
-    const numRecapItems = resp.aggregations.item_location._nested.buckets
-      .filter((bucket) => /^loc:rc/.test(bucket.key))
-      .reduce((sum, entry) => entry.doc_count + sum, 0)
+    // Get total number of items:
+    const numItems = (resp.hits.hits[0]._source.numItemsTotal || resp.hits.hits[0]._source.numItems)[0]
+
+    // For partner bibs, all items are ReCAP items:
+    const numRecapItems = nyplSource !== 'sierra-nypl'
```

Comment:
is this where the bug was originating? Were we not getting an accurate count for partner items?